lib/java/com/google/android/material/button/MaterialButton.java

```diff
@@ -157,8 +157,22 @@
    */
   public static final int ICON_GRAVITY_TEXT_END = 0x4;
 
+  /**
+   * Gravity used to position the icon in the center of the view on top of the text
```

Comment:
It is not really on top of the text.
Should be: `Gravity used to position the icon at the top of the view.`

Reply:
Fixed by be01462